projects/Mallard/src/AppNavigation.tsx

```diff
@@ -174,6 +176,7 @@ const MainStack = () => {
 		<Main.Navigator
 			initialRouteName={RouteNames.Home}
 			screenOptions={{
+				cardStyle: { backgroundColor: neutral[100] },
```

Comment:
Do we have a theme value we can use here (i.e `color.Background` from `color.ts` ) rather than hardcoding a specific value?

Reply:
Nothing for this one from source but I'll add it in 👍